z80.hpp

```diff
@@ -5084,6 +5084,9 @@ class Z80
         this->CB.out = out;
         this->CB.arg = arg;
         ::memset(&reg, 0, sizeof(reg));
+        reg.pair.A = 0xff;
+        reg.pair.F = 0xff;
+        reg.SP = 0xffff;
```

Comment:
Thanks for the information!
I will merge it after confirmation.

![image](https://user-images.githubusercontent.com/7346381/154823509-798f4a53-3dba-4f8b-a68a-df329b2ef087.png)